content/blog/2017/2017-04-05-welcome-to-blue-ocean.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Getting Started with Blue Ocean"
+tags:
+- blueocean
+- ux
+- pipeline
+
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:https://cloudbees.com[CloudBees].
+
+== Welcome to Blue Ocean 1.0!
+
+In case you've been heads down on other projects
+link:/blog/2016/05/26/introducing-blue-ocean/[for the past 10 months],
+Blue Ocean is a new user experience for Jenkins,
+and version 1.0 was released today!
+Blue Ocean takes Jenkins beyond of the realm of infrastructure tools used only by specialists,
+and makes Jenkins and continuous delivery approachable to every user on any size team.
+I've been working with it for the past several months, and I can tell you it is amazing.
+I wish all the interactions with Jenkins were as easy as this:
+
+image::content/doc/book/resources/blueocean/intro/new-pipeline-box.png[It's time to create your first Pipeline!, role=center]
+
+== 10 minutes to Blue Ocean
+
+Blue Ocean is simple to install and will work on basically any Jenkins 2 instance (version 2.7 or later).
+Even better, it runs side-by-side with the existing Jenkins web UI -
+you can switch back and forth between them whenever you like.
+There's really no risk.
+If you have a Jenkins instance and a good network connection,
+in 10 minutes you could be using Blue Ocean.
+
+. Login to your Jenkins server
+. Click **Manage Jenkins** in the sidebar then **Manage Plugins**
+. Choose the **Available** tab and use the search bar to find **Blue Ocean**
+. Click the checkbox in the Install column
+. Click either **Install without restart** or **Download now and install after
+restart**
+
+image::content/doc/book/resources/blueocean/intro/plugin-install.png[Installing Blue Ocean, role=center]
+
+After you install Blue Ocean, you can start using it
+by clicking on **Open Blue Ocean** in the top navigation bar of the
+Jenkins web UI, or you can navigate directly to Blue Ocean by adding
+`/blue` to your Jenkins URL, for example `https://ci.jenkins.io/blue`.
+
+image::content/doc/book/resources/blueocean/intro/switch-blue-ocean.png[Opening Blue Ocean, role=center]
+
+If you have to go back to the "classic" Jenkins UI,
+there's an "exit" icon located at the top of most pages in Blue Ocean.
+
+image::content/doc/book/resourcesblueocean/intro/switch-classic.png[Returning to the "classic" web UI, role=center]
+
+=== Dive in!
+
+That's it! You now have a working Blue Ocean installation.
+Take a look around at your Pipelines and activity, or try creating a new Pipeline.
+I think you'll be pleasantly surprised at how intuitive and helpful Blue Ocean can be.
+My main complaint about Blue Ocean is that I still have to go back to the old Jenkins web UI for some tasks.
```

Comment:
Not sure I understand why your complaint is necessary here :smiling_imp: 

I think you're trying to convey a positive, but it reads rather negative to me like "Blue Ocean isn't done yet" rather than "I like this so much I never want to leave"